ckan/logic/action/get.py

```diff
@@ -1444,13 +1444,13 @@ def user_show(context: Context, data_dict: DataDict) -> ActionResult.UserShow:
     else:
         user_obj = None
 
-    if not user_obj:
-        raise NotFound
-
-    context['user_obj'] = user_obj
+    if user_obj:
+        context['user_obj'] = user_obj
```

Comment:
When updating this PR to target master, I found that the Context object no longer accepts None for the 'user_obj' property, so I've added this simple check. This is untested as I don't have a CKAN instance running on `master` locally.